pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```
@@ -13,10 +13,13 @@
 , indilib
 , libgphoto2
 , libraw
+, libraw1394
 , libftdi1
 , libdc1394
 , gpsd
 , ffmpeg
+, udev
```

```suggestion
, libftdi1
, libdc1394
, gpsd
, ffmpeg
```

(see the other comments)